docs/content/latest/explore/multi-region-deployments/read-replicas.md

```diff
@@ -0,0 +1,281 @@
+./bin/yb-ctl add_node --placement_info "c.r2.z23" --tserver_flags "placement_uuid=rr"
+
+./bin/yb-admin -master_addresses 127.0.0.1:7100,127.0.0.2,127.0.0.3 add_read_replica_placement_info c.r2.z21:1,c.r2.z22:1,c.r2.z23:1 3 rr
+```
+
+The following illustration demonstrates the setup of two clusters, one of which is primary and another one is read replica visible via [Yugabyte Platform](https://docs.yugabyte.com/latest/yugabyte-platform/):
+
+![img](/images/explore/multi-region-deployments/read-replicas4.png)
+
+The following command directs `CL.ONE` reads to the primary cluster (as follower reads) in region `r`:
+
+```shell
+java -jar ./yb-sample-apps.jar --workload CassandraKeyValue \ 
+                               --nodes 127.0.0.1:9042,127.0.0.2:9042,127.0.0.3:9042 \ 
+                               --nouuid \ 
+                               --num_unique_keys 2 \ 
+                               --num_threads_write 1 \ 
+                               --num_threads_read 1 \ 
+                               --value_size 1024 --local_reads --with_local_dc r
+```
+
+The following illustration demonstrates the result of exectuting the preceding command (visible via [Yugabyte Platform](https://docs.yugabyte.com/latest/yugabyte-platform/):):
+
+![img](/images/explore/multi-region-deployments/read-replicas5.png)
+
+The following command directs the `CL.ONE` reads to the read replica cluster (as observer reads) in region `r2`:
+
+```shell
+java -jar ./yb-sample-apps.jar --workload CassandraKeyValue \ 
+                               --nodes 127.0.0.1:9042,127.0.0.2:9042,127.0.0.3:9042 \ 
+                               --nouuid \ 
+                               --num_unique_keys 2 \ 
+                               --num_threads_write 1 \ 
+                               --num_threads_read 1 \ 
+                               --value_size 1024 --local_reads --with_local_dc r2
+```
+
+The following illustration demonstrates the result of exectuting the preceding command (visible via [Yugabyte Platform](https://docs.yugabyte.com/latest/yugabyte-platform/)):
+
+![img](/images/explore/multi-region-deployments/read-replicas6.png)
+
+## Fault Tolerance
+
+In the strong consistency mode (default), more failures can be tolerated by increasing the number of replicas: to tolerate a `k` number of failures, `2k+1` replicas are required in the RAFT group. However, follower reads and observer reads can provide Cassandra-style `CL.ONE` fault tolerance. The  `max_stale_read_bound_time_ms` GFlag controls how far behind the followers are allowed to be before they redirect reads back to the RAFT leader (the default is around 60). For "write once, read many times” workloads, this number could be increased. By stopping nodes, you can induce behavior of follower and observer reads.
```

Comment:
"(the default is around 60)" -> "(the default is 60 seconds)"

"By stopping nodes, you can induce behavior of follower and observer reads" -> something else
We're trying to demonstrate that even after killing two nodes you are able to continue to read. Without follower reads this is not possible.

Reply:
1) Changed.
2) Changed to "By stopping nodes, you can induce behavior of follower and observer reads such that they continue to read (which would not be possible without follower reads)."